src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -25,6 +32,13 @@
   public void contextInitialized(ServletContextEvent sce) {
     try {
       List<User> users = PersistentStorageAgent.getInstance().loadUsers();
+
+      String adminPassword = "admin";
```

Comment:
Nit: I would extract this hardcoded username and password into constants--make them private static final and put them at the top of the file. That way they're easier to find by looking at the beginning of this class, rather than buried in the method implementation. (https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)